lib/zendesk_api/error.rb

```diff
@@ -2,13 +2,17 @@ module ZendeskAPI
   module Error
     class ClientError < Faraday::Error::ClientError
       attr_reader :wrapped_exception
+
+      def to_s
+        "#{super} -- #{@response.method} #{@response.url}"
```

Comment:
Actually, `Faraday::Error::ClientError` also defines an `attr_reader :response`. So for consistency, you can just use `response` here as well.

https://github.com/lostisland/faraday/blob/master/lib/faraday/error.rb#L6